web/src/features/fireWeather/components/graphs/plotlyHelper.ts

```diff
@@ -66,29 +66,36 @@ interface TempRHValue {
   datetime: string
   temperature?: number | null
   relative_humidity?: number | null
+  dewpoint?: number | null
   tmp_tgl_2_5th?: number
   tmp_tgl_2_90th?: number
   rh_tgl_2_5th?: number
   rh_tgl_2_90th?: number
-  tmp_max?: number
-  tmp_min?: number
+  tmp_dp_max?: number
+  tmp_dp_min?: number
```

Comment:
Don't we need `tmp_max` and `tmp_min` for Forecast?

Reply:
Well, the variable names are confusing. It implies that it has something to do with dew points no? also there is no `tmp_dp_max` property in any weather data type, which means line 173 `if (tmp_dp_min && tmp_dp_max) {` will never get executed

Reply:
I should note that `tmp_max` and `tmp_min` are from ForecastSummary which are used to draw vertical lines. And I see why we didn't see any tests failing since we are not testing this part.